packages/data-context/src/sources/MigrationDataSource.ts

```diff
@@ -164,8 +176,12 @@ export class MigrationDataSource {
     try {
       const supportFiles = await supportFilesForMigration(this.ctx.currentProject)
 
+      debug('supportFilesForMigrationGuide: supportFiles %O', supportFiles)
+
       return supportFiles
-    } catch {
+    } catch (err) {
+      debug('supportFilesForMigrationGuide: err %O', err)
```

Comment:
I like this, if we're doing nothing with the error then at least lets get a debug in there to make troubleshooting easier in the future